### DISCUSSION TITLE
Maintaining a OpenCv build repository for supporting robotgo's functionality.

### DISCUSSION THREAD

**Discussion by SakthiMahendran** [go-vgo/robotgo#468] [General]
We know that robotgo supports OpenCv's functionality through gocv.

The problem is that it is too difficult to build OpenCv for different platforms.

if we want to deploy our app on different os since golang supports cross compilation I can write my code in windows and compile to linux and try to distribute it.

But if I am using opencv in my app
I need a Linux build for OpenCv.

Where do I get it?

It is huge pain right.

So i would like to suggest to maintain a OpenCv build repository for different os.

Note: We can't use OpenCv's official build with gocv. So we need a seperate build repo.

**Comment by vcaesar** [comment]
I will think about it, maybe build like bitmap, but the time is limit.
